services/terrain/src/terrain/clients/iplant_groups.clj

```diff
@@ -10,6 +10,7 @@
   {:username (:id response)
    :firstname (:first_name response)
    :lastname (:last_name response)
+   :name (:name response)
```

Comment:
I added the user's full name (from the `CN` field in LDAP) to the response so that we can avoid having to reassemble the user's name.